### PR TITLE
Docs: edited the tx, block and vote models section

### PR DESCRIPTION
**Pull request by ttmc**
The whitepaper includes a more-or-less verbatim copy of "The Transaction, Block and Vote Models" section of the BigchainDB docs.

I'm updating the whitepaper so I need to update that section. As part of that, I edited the corresponding section of the docs.

(I'm not sure it makes sense to have so much detail in the whitepaper, but that's a separate issue.)

It seems we could use more documentation on conditions and fulfillments, but maybe we can do that when we document examples of using the forthcoming asset models, or the forthcoming improved client API.

@diminator I'd like you to review these edits before I merge them, since the main new stuff is related to the conditions and fulfillments.

### CODE REVIEW

File: docs/source/models.md [hunk 1]
```diff
@@ -6,81 +6,86 @@ Transactions, blocks and votes are represented using JSON documents with the fol
...
 ```json
 {
-    "id": "<sha3 hash>",
-    "version": "<transaction version number>",
+    "id": "<SHA3-256 hash hexdigest of transaction (below)>",
+    "version": "<version number of the transaction model>",
     "transaction": {
-        "fulfillments": ["<list of <fullfillment>"],
-        "conditions": ["<list of <condition>"],
+        "fulfillments": ["<list of fullfillments>"],
```
Comment (diminator) [inline docs/source/models.md]: fulfillments (less l's)

File: docs/source/models.md [hunk 2]
```diff
@@ -6,81 +6,86 @@ Transactions, blocks and votes are represented using JSON documents with the fol
...
             "hash": "<SHA3-256 hash hexdigest of payload>",
-            "payload": "<generic json document>"
+            "payload": "<any JSON document>"
         }
     }
 }
 ```
 
-A transaction is an operation between the `current_owner` and the `new_owner` over the digital content described by `hash`. For example if could be a transfer of ownership of the digital content `hash`
+Transactions are the basic things stored by BigchainDB. There are two kinds:
```
Comment (diminator) [inline docs/source/models.md]: things > records?

File: docs/source/models.md [hunk 3]
```diff
@@ -6,81 +6,86 @@ Transactions, blocks and votes are represented using JSON documents with the fol
...
+    - `public_key`: The _new_owner's_ public key.
+    - `type_id`: The fulfillment type ID; see the [ILP spec](https://interledger.org/five-bells-condition/spec.html).
+    - `uri`: Binary representation of the condition using only URL-safe characters.
 
-##### Multi Signature
+#### Multiple New Owners
 
-If there are multiple _new owners_ by default we create a condition requiring a signature from each new owner in order
-to spend the digital asset.
+If there are multiple _new owners_, we can create a condition requiring a signature from each new owner in order
```
Comment (diminator) [inline docs/source/models.md]: create a ThresholdCondition

File: docs/source/models.md [hunk 4]
```diff
@@ -121,16 +126,17 @@ Example of a condition with two _new owners_:
     - `weight`: integer weight for each subfulfillment's contribution to the threshold
```
Comment (diminator) [inline docs/source/models.md]: I have a feeling we are replicating the docs of crypto-conditions/ILP spec

These (weight, threshold) are attributes of a ThresholdCondition

File: docs/source/models.md [hunk 5]
```diff
@@ -140,19 +146,19 @@ If there is only one _current owner_ the fulfillment will be a single signature
 }
 ```
 
-- `fid`: Fulfillment index. It matches a `cid` in the conditions with a new _crypto condition_ that the new owner(s)
-need to fulfill to spend this digital asset
-- `current_owners`: Public key of the current owner(s)
+- `fid`: Fulfillment index. It matches a `cid` in the conditions with a new _crypto condition_ that the new owner
+  needs to fulfill to spend this asset.
+- `current_owners`: A list of public keys of the current owners; in this case it has just one public key.
 - `fulfillment`:
```
Comment (diminator) [inline docs/source/models.md]: fulfillment: a cryptoconditions URI that encodes the cryptographic fulfillments like signatures and others, see crypto-conditions

**Comment by ttmc** [comment]
@diminator I made some edits based on your line notes.

Unless there are any objections in the next day or so, I will merge this PR.

**Pull request merged**